TramsDataApi/ResponseModels/AcademyConversionProject/AcademyConversionProjectResponse.cs

```diff
@@ -80,7 +81,10 @@ public class AcademyConversionProjectResponse
         public int? YearThreeProjectedPupilNumbers { get; set; }
         public string SchoolPupilForecastsAdditionalInformation { get; set; }
 
-        //key stage performance tables
+        // key stage performance tables
         public string KeyStagePerformanceTablesAdditionalInformation { get; set; }
+
+        // project notes
+        public List<ProjectNoteResponse> ProjectNotes { get; set; }
```

Comment:
Similar to above. I think either have a separate GET method that returns the list of project notes or maybe a query string param to indicate include project notes in the response as they don't appear in the task list only the project notes tab